sdks/java/core/src/main/java/org/apache/beam/sdk/io/Compression.java

```diff
@@ -152,6 +156,38 @@ public WritableByteChannel writeCompressed(WritableByteChannel channel) throws I
     }
   },
 
+  /** LZO compression. */
+  LZO(".lzo", ".lzo") {
```

Comment:
Should this use the `.lzo_deflate` extension for suggested and detected extensions?

Reply:
Thanks for pointing this out. I have made the appropriate changes on this.